### PR TITLE
Updated php image to 8.3

### PR DESCRIPTION
**Pull request by temlife**
Use this PR as patch to check custom project compatibility with PHP 8.2

`curl https://github.com/skilld-labs/skilld-docker-container/pull/435.diff | patch -p1`

Blockers

- [x]  https://www.drupal.org/project/drupal/issues/3328005
- [x]  https://www.drupal.org/project/webform/issues/3329579

### CODE REVIEW

**Comment by andypost** [comment]
I bet it's too early to move to 8.2 as there's few core issues and I bet contrib/custom code

File: README.md [hunk 1]
```diff
@@ -59,7 +59,7 @@
 | SITE_MAIL  | Site e-mail address | admin@example.com |
 | ADMIN_NAME  | Admin username | admin |
 | PROJECT_INSTALL | Way to install site - from straight or existing config | - |
-| IMAGE_PHP | Php image to use | `skilldlabs/php:81-fpm` |
+| IMAGE_PHP | Php image to use | `skilldlabs/php:82-unit` |
```
Comment (andypost) [inline README.md:62]: I pushed the change as 94a071e5ac1e66988e65968f1837c9f03519755b

Comment (andypost) [reply README.md:62]: please rebase and squash commits

**Comment by andypost** [comment]
This can be used to check project compatibility with PHP 8.2 via

`curl https://github.com/skilld-labs/skilld-docker-container/pull/435.diff | patch -p1`

File: README.md (hunk 1 above)
Comment (temlife) [reply README.md:62]: Rebased, squashed @andypost

Comment (andypost) [reply README.md:62]: Thank you!

**Comment by andypost** [comment]
Let's wait committing it because (at least)
- https://www.drupal.org/project/drupal/issues/3328005
- https://www.drupal.org/project/webform/issues/3329579

**Comment by andypost** [comment]
https://www.youtube.com/watch?v=z0Tzb6SVwr4

nice example of upgrade

File: .env.default [hunk 2]
```diff
@@ -7,7 +7,7 @@ SITE_MAIL=admin@example.com
 ADMIN_NAME=admin
 ADMIN_MAIL=admin@example.com
 PROJECT_INSTALL=
-IMAGE_PHP=skilldlabs/php:81-unit
+IMAGE_PHP=skilldlabs/php:82-unit
```
Comment (andypost) [inline .env.default:10]: As 10.2 is out, should be 8.3

**Review by andypost — CHANGES_REQUESTED** [review]
needs work to upgrade to 8.3

File: .env.default (hunk 2 above)
Comment (andypost) [reply .env.default:10]: and https://github.com/skilld-labs/skilld-docker-container/issues/448

Comment (andypost) [reply .env.default:10]: It also needs to update `Makefile` for extra extensions

Comment (andypost) [reply .env.default:10]: Let's go with 8.2 as NewRelic has no release for PHP 8.3

**Comment by andypost** [comment]
@temlife please rebase

**Comment by andypost** [comment]
@temlife please update it to `php:83` https://www.drupal.org/project/drupal/issues/3330874

**Comment by andypost** [comment]
Closing in favour of https://github.com/skilld-labs/skilld-docker-container/pull/457

**Pull request closed without merging**